components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -387,6 +394,72 @@ - (CGFloat)estimatedTextViewLineHeight {
   return MDCCeil(self.textView.font.lineHeight * 2.f) / 2.f;
 }
 
+#pragma mark - Trailing View Implementation
+
+- (void)updateTrailingViewLayout {
+  if (!self.trailingView) {
+    return;
+  }
+
+  self.trailingView.translatesAutoresizingMaskIntoConstraints = NO;
```

Comment:
I would only expect to set this once, when trailingView is created or added.
There may be a deeper reason I'm not seeing.  If so, perhaps a comment.